gencrud/config/references.py

```diff
@@ -22,9 +22,9 @@
 
 
 class TemplateReferences( TemplateBase ):
-    def __init__( self, cfg ):
+    def __init__( self, **cfg ):
```

Comment:
is the replacement of certain parameters by their asteriks, i.e., **-counterpart, a change that should also affect the Github gencrud version?